plugins-bundled/internal/plugin-catalog-app/src/RootPage.tsx

```diff
@@ -1,15 +1,71 @@
 import { AppRootProps } from '@grafana/data';
-import { pages } from 'pages';
 import React from 'react';
+import { css } from 'emotion';
+import { Discover } from 'pages/Discover';
+import { Browse } from 'pages/Browse';
+import { PluginDetails } from 'pages/PluginDetails';
+import { Library } from 'pages/Library';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 export const MarketplaceRootPage = React.memo(function MarketplaceRootPage(props: AppRootProps) {
-  const {
-    path,
-    query: { tab },
-  } = props;
-  // Required to support grafana instances that use a custom `root_url`.
-  const pathWithoutLeadingSlash = path.replace(/^\//, '');
-
-  const Page = pages.find(({ id }) => id === tab)?.component || pages[0].component;
-  return <Page {...props} path={pathWithoutLeadingSlash} />;
+  return (
+    <div
```

Comment:
Shouldn't this be handled by the `Page` component or something that can be reused by other apps?